vllm/entrypoints/neuron_multi_node/api_server.py

```diff
@@ -0,0 +1,172 @@
+"""
```

Comment:
Does it make sense to refactor this + the base `api_server.py` so we have something we can deduplicate this and extend in the future?

Reply:
Done.